src/CraigslistAPI/CraigslistSearchComponent.js

```diff
@@ -95,7 +96,11 @@ export default class CraigslistSearchComponent extends React.Component {
                     )}
                 </div>
             }
-            
+            <div>
+                <a href={"../privacypolicy"}>Privacy Policy</a>
+            </div>
```

Comment:
We should put this in the FooterComponent  and add the CSS file to that component as well